workflow/rocoto/gfs_tasks.py

```diff
@@ -1897,21 +1897,24 @@ def fit2obs(self):
 
     def metp(self):
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.run}_arch'}
+        if self.options['do_archtar']:
+            dep_dict = {'type': 'task', 'name': f'{self.run}_arch_tars'}
+            deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
The `arch_tars` job shouldn't be a dependency for the `metp` jobs.  I don't think you have to rerun your local test after you fix this.